crates/networking/p2p/sync.rs

```diff
@@ -76,7 +83,9 @@ pub enum SyncMode {
 /// Only performs full-sync but will also be in charge of snap-sync in the future
 #[derive(Debug)]
 pub struct SyncManager {
```

Comment:
maybe SyncTask would be more appropriate?

Reply:
The way I think about it is that the `SyncManager` is in charge of performing the sync itself while the `SyncSupervisor` is the middleman between the sync manager and the rest of the codebase, but I think `SyncManager` and `Syncer` are appropriate names

Reply:
Updated [e2c3279](https://github.com/lambdaclass/ethrex/pull/2303/commits/e2c32799621b53d5b40c784c04102dc96a6bc82d)